operations/enterprises.md

```diff
@@ -297,163 +300,295 @@ Returns all spaces of an enterprise associated with the connector integration.
-| `SpaceFeatureAssignments` | array of [Space feature assignment](enterprises.md#space-feature-assignment) | optional | Assignments of space features to spaces. |
+| `Resources` | array of [Resource](#resource) | required | The resources of the enterprise. |
+| `ResourceCategories` | array of [Resource category](#resource-category) | required | Categories of resources in the enterprise. |
+| `ResourceCategoryAssignments` | array of [Resource category assignment](#resource-category-assignment) | optional | Assignments of resources to categories. |
+| `ResourceCategoryImageAssignments` | array of [Resource category assignment](#resource-category-image-assignment) | optional | Assignments of images to categories. |
+| `ResourceFeatures` | array of [Resource feature](#resource-feature) | optional | Features of resources in the enterprise. |
+| `ResourceFeatureAssignments` | array of [Resource feature assignment](#resource-feature-assignment) | optional | Assignments of resource features to resources. |
 
-#### Space
+#### Resource
 
 | Property | Type |  | Description |
 | --- | --- | --- | --- |
-| `Id` | string | required | Unique identifier of the space. |
-| `IsActive` | bool | required | Whether the space is still active. |
-| `Type` | string [Space type](enterprises.md#space-type) | required | Type of the space. |
-| `Number` | string | required | Number of the space \(e.g. room number\). |
-| `FloorNumber` | string | optional | Number of the floor the space is on. |
-| `BuildingNumber` | string | optional | Number of the building the space is in. |
-| `ParentSpaceId` | string | optional | Identifier of the parent [Space](enterprises.md#space) \(e.g. room of a bed\). |
-| `CategoryId` | string | required | Identifier of the [Space category](enterprises.md#space-category) assigned to the space. |
-| `State` | string [Space state](enterprises.md#space-state) | required | State of the room. |
-
-#### Space type
-
-* `Room`
-* `Dorm`
-* `Bed`
-* ...
+| `Id` | string | required | Unique identifier of the resource. |
+| `IsActive` | bool | required | Whether the resource is still active. |
+| `Name` | string | required | Name of the resource \(e.g. room number\). |
+| `ParentResourceId` | string | optional | Identifier of the parent [Resource](#resource) \(e.g. room of a bed\). |
+| `State` | string [Resource state](#resource-state) | required | State of the resource. |
+| `Data` | [Resource data](#resource-data) | required | Additional data of the resource. |
 
-#### Space state
+#### Resource state
 
 * `Dirty`
 * `Clean`
 * `Inspected`
 * `OutOfService`
 * `OutOfOrder`
 
-#### Space category
+#### Resource data
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Discriminator` | string [Resource data discriminator](#resource-data-discriminator) | required | If resource is space, object or person. |
+| `Value` | object | required | Based on Resource data discriminator, e.g. [Space resource data](#space-resource-data)  |
```

Comment:
add link to RDD

Reply:
link is one line above and it is written in the same way as Tax rate strategy
![image](https://user-images.githubusercontent.com/24473668/86088809-4ea59280-baa7-11ea-89ae-6d011ef2a594.png)